### PR TITLE
Add slotchange event.

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: api/slotchange.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
```
Comment (chrisdavidmills) [inline api/slotchange.json]: Flagging @Elchi3 here — is this the correct way to add an actual event to BCD (as opposed to a DOM event object?)

File: api/slotchange.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  "api": {
+    "slotChange": {
```
Comment (chrisdavidmills) [inline api/slotchange.json]: slotchange?

File: api/slotchange.json (hunk 1 above)
Comment (chrisdavidmills) [reply api/slotchange.json]: I mean, shouldn't we put events in a separate events folder? And maybe have the JSON structure as events.slotchange or similar?

Comment (Elchi3) [reply api/slotchange.json]: Yes, please hold on with events for now. See the discussion in https://github.com/mdn/browser-compat-data/issues/935.

**Comment by Elchi3** [comment]
Closing this for now

**Pull request closed without merging**